Gemfile

```diff
@@ -4,6 +4,9 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby "2.7.0"
 
 gem "dotenv-rails", groups: [:development, :test]
+gem "faker"
+gem "will_paginate", "3.1.5"
+gem "bootstrap-will_paginate", "1.0.0"
```

Comment:
Please, change to gem kaminari instead of will_paginate
https://github.com/kaminari/kaminari